drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -99,5 +102,16 @@ def enrollment_access_summaries(filters: nil)
           project_ids: filters&.project,
         )
     end
+
+    def staff_assignments
+      load_ar_association(object, :staff_assignments).
```

Comment:
This could be a performance concern for queries that resolve staff_assignments on a collection of users. But we are probably only resolving this on one user at a time in practice.

So the simplest thing to do would probably be to just skip using the data loader in this case.
```suggestion
      # n+1 queries, needs optimization to be performant on collections
      object.staff_assignments.
```

Reply:
That makes sense, thank you for the explanation!